app/budget/directives/courseCosts/courseCosts.html

```diff
@@ -1,23 +1,19 @@
 <div class="course-costs">
-	<term-selector
+	<!-- <term-selector
 		selected-term="selectedBudgetScenario.selectedTerm"
 		terms="selectedBudgetScenario.terms"
 		term-descriptions="selectedBudgetScenario.termDescriptions">
-	</term-selector>
+	</term-selector>-->
 
```

Comment:
I think the term selector is important, what was the reason behind commenting it out?